packages/katello/katello/katello-remove

```diff
@@ -132,61 +142,73 @@ CONTENT=[
   '/opt/theforeman',
   '/var/www/html/pub/katello-rhsm-consumer',
   '/var/tmp/foreman-proxy'
-]
+].freeze
 
 CONTENT.map! do |dir|
-  if File.readlines('/proc/mounts').any?{ |line| line.split(" ")[1] =~ /#{dir}$/ }
+  if File.readlines('/proc/mounts').any? { |line| line.split(' ')[1] =~ /#{dir}$/ }
     dir + '/*'
   else
     dir
   end
 end
 
 def remove
+  puts 'Starting removal'
+  puts 'Stopping services'
   `katello-service stop`
 
-  puts "Removing RPMs"
+  puts 'Removing RPMs'
   packages = []
-  RPMS.each do | rpm |
+  RPMS.each do |rpm|
     rpmpackages = `rpm -qa | grep "#{rpm}"`
     packages += rpmpackages.lines.map(&:chomp)
   end
-  `yum erase -y #{packages.join(" ")} > /dev/null 2>&1`
+  `yum erase -y #{packages.join(' ')} > /dev/null 2>&1`
 
-  puts "Cleaning up configuration files"
+  puts 'Cleaning up configuration files'
   FileUtils.rm_rf CONFIG_FILES
 
-  puts "Cleaning up log files"
+  puts 'Cleaning up log files'
   # logs
   FileUtils.rm_rf LOG_FILES
 
-  puts "Cleaning up Certs"
+  puts 'Cleaning up certs'
   # pulp cert stuff
   FileUtils.rm_rf CERT_FILES
 
-  puts "Cleaning up content"
-  #content
+  puts 'Cleaning up content'
+  # content
   FileUtils.rm_rf CONTENT
+
+  puts 'Finished'
+end
+
+def check
+  puts "\nWARNING: This script will erase many packages and config files."
+  puts "Important packages such as the following will be removed:\n"
+  puts '  * httpd (apache)'
+  puts '  * mongodb'
+  puts '  * tomcat'
+  puts '  * puppet'
+  puts '  * ruby'
+  puts '  * rubygems'
+  puts "  * All Katello and Foreman Packages\n"
+  puts 'Once these packages and configuration files are removed there is no going back.'
+  puts 'If you use this system for anything other than Katello and Foreman you probably'
+  puts "do not want to execute this script.\n"
+  puts "\nRead the source at /usr/sbin/katello-remove for a list of what is removed.  Are you sure(Y/N) "
+
+  response = gets.chomp
+  if /[Y]/i.match(response)
```

Comment:
if I am reading this correctly, this can be written as

```
if /[Y]/i.match(response) && confirmed?
  remove
else
  puts '**** cancelled ****'
end
```